docs/system_overview/troubleshooting.md

```diff
@@ -175,6 +175,9 @@ In iRODS 4.3.0, log output is managed via **server_config.json**. The server now
         // Controls log output related to the Delay Server process.
         "delay_server": "info",
 
+        // Controls log output related to the GenQuery2 parser.
+        "genquery2": "info",
```

Comment:
@trel Notice the comment uses the word parser rather than API. That's intentional because that log category controls whether the actual parser used within the GenQuery2 API outputs log messages. For API activity, the api category would be used.

It may be a good idea to mention something about that. Thoughts?

Reply:
I think I'm going to add some words about that so it's clear what the category is for.

Reply:
Agreed.

Reply:
Done.